zio-http/shared/src/main/scala/zio/http/Header.scala

```diff
@@ -2543,12 +2544,21 @@ object Header {
   }
 
   object ContentType extends HeaderType {
+    private final val CacheInitialSize = 256
```

Comment:
Is this not already big? Most servers will probably see 10 content types max I would assume.

Reply:
We can lower it, although I don't think there is much benefit in making it smaller memory-wise since this is initialized just once. Plus HashMaps with a larger capacity have a better read performance due to a better hash key distribution

Reply:
I'll lower it to 64 though cause 256 does seem a bit extensive